src/Types/Redeemer.purs

```diff
@@ -0,0 +1,18 @@
+module Types.Redeemer
+  ( unitRedeemer
+  ) where
+
+import Types.PlutusData (PlutusData(..))
+import Types.RedeemerTag (RedeemerTag)
+import Types.Transaction (Redeemer(..))
+import Data.BigInt as BigInt
+
+-- FIX ME (create an issue unless someone notices simple solution to this in PR
+-- review)
+unitRedeemer :: RedeemerTag -> Redeemer
```

Comment:
I don't think this is what we want. `unitRedeemer` from Plutus is just: `Redeemer $ toBuiltinData ()`. The only field we could possibly specify here for a `Types.Transaction.Redeemer` would be `data`. We can't know its index, ex units, and tag without knowing which input the redeemer belongs to

Reply:
Yes, so `Redeemer $ toData unit` should be fine. Just to be clear @klntsky, Plutus has extra features for `BuiltInData` which is a wrapper over `Data` (our `PlutusData`) which we can ignore, we can essentially treat it as `toData`.